### ISSUE TITLE
Biopython can't deal with PDB File with len(Chain ID) > 1?

### ISSUE THREAD

**Issue by Kirito-Ausna** [biopython/biopython#4179]
### Setup

Hello! Thanks for your awesome work for biodata process. Recently I used biopython to process the Enzyme Commission Dataset From _Structure-based protein function prediction using graph convolutional networks(https://www.nature.com/articles/s41467-021-23303-9), I find that biopython will fail in case of PDB Files with len(Chain ID) > 1, such as chain id is AA, D1, AAA. etc.
I find in the source code you get chain id with chainid = line[21], and actually there exists PDB File with len(Chain ID) > 1, so I wonder what's the reason for you to assign chain id with just one letter. I can just adopt these abnormal files, but I am curious about how I can process these files.

**Comment by peterjc** [comment]
This is almost certainly following the PDB file format specification, in this case a little too strictly for your files.

**Comment by rob-miller** [comment]
Yes, for PDB files the chain ID is only a single character in column 22 of
the ATOM record as described here:

https://www.wwpdb.org/documentation/file-format-content/format33/sect9.html#ATOM

To process your files one option is to use mmCIF format.

**Comment by Kirito-Ausna** [comment]
Thanks! I appreciate your awesome work!
